src/Microsoft.Extensions.Logging.Abstractions/LoggerMessage.cs

```diff
@@ -0,0 +1,343 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging.Internal;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.Logging
+{
+    /// <summary>
+    /// Creates delegates which can be later cached to log messages in a performant way.
+    /// </summary>
+    public static class LoggerMessage
+    {
+        /// <summary>
+        /// Creates a delegate which can be used for creating a log scope.
```

Comment:
`can be used for creating a log scope` -> `can be invoked to create a log scope`